pages/dev/p/_projectname.vue

```diff
@@ -128,24 +137,43 @@ export default {
     this.projectName = this.$route.params.projectname;
   },
   mounted() {
-    //let projectObj = axiosCallToDatabase();
-    let projectObj = {
-      name: "Aceable",
-      numSubs: 1000,
-      projectId: '1111',
-      pictureURL: '',
-    }
-
-    // get info from obj
-    this.name = projectObj.name;
-    this.pictureURL = projectObj.pictureURL;
-    this.numSubs = projectObj.numSubs;
-    this.projectId = projectObj.projectId;
-
-    // get info from state
-    this.isSubscribed = this.$store.getters['user/isUserSubscribed'];
     this.isOwner = this.$store.getters['user/isUserOwner'];
+    this.isSubscribed = this.$store.getters['user/isUserSubscribed'];
 
+    this.$axios.get(`/api/v1/projects/p/${this.projectName}`)
+      .then(({data}) => {
+        if (data.hasOwnProperty('headerPicture')) {
```

Comment:
Make a note here in code that this is just for development until we're rock solid on our data